admin-dev/themes/new-theme/js/pages/product/edit/specific-prices-manager.ts

```diff
@@ -79,6 +82,47 @@ export default class SpecificPricesManager {
     });
   }
 
+  private initCatalogPriceRules() {
+    const catalogPriceRuleList = new CatalogPriceRuleList();
+
+    const showCatalogPriceRulesButton = document.querySelector<HTMLElement>(CatalogPriceRulesMap.showCatalogPriceRules);
+    const hideCatalogPriceRulesButton = document.querySelector<HTMLElement>(CatalogPriceRulesMap.hideCatalogPriceRules);
```

Comment:
To confirm with @MatShir but I wouldn't handle a toggle for this component, it should be hidden by default to improve performance and avoid a systematic API call on the page. It's only shown when we click on the button, but then it always stays visible, so the show button disappears and no need for a Hide button